src/module/postgame.js

```diff
@@ -0,0 +1,62 @@
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const submit = document.getElementById('submit');
+const ul = document.getElementById('ul');
+const refresh = document.getElementById('refresh');
+
+export const getGameId = localStorage.getItem('gameId');
+
+export const postData = (gameId) => {
+  const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`;
+  return fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name.value,
+      score: score.value,
+    }),
+  })
+    .then((response) => response.json())
+    .catch((error) => {
+      throw error;
+    });
+};
+
+submit.addEventListener('click', () => {
+  postData(getGameId);
+});
+
+export const getData = (getGameId) => {
+  const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${getGameId}/scores`;
+  return fetch(url)
+    .then((response) => response.json())
+    .catch((error) => {
+      throw error;
+    });
+};
```

Comment:
[OPTIONAL] I recommend fetching data immediately after the app opens since sometimes data already exits and we want users to see that in order to improve UX